source/_templates/versioning.html

```diff
@@ -0,0 +1,34 @@
+{%- if versions %}
+{% if html_theme == 'sphinx_rtd_theme' %}
+<span class="rst-widget-header">
+    <span class="fa fa-book rst-widget-header-left"> Other Versions</span>
+    v: {{ current_version.version }}
+    <span class="fa fa-caret-down"></span>
+</span>
+<div class="rst-widget-body">
+    {%- if versions.in_development %}
+    <dl>
+        <dt>Development Versions</dt>
```

Comment:
If it's easy, could you skip making `current_version.version` a link and bold the text to indicate it's the current selection? Nevermind if it's a pain.